core/src/main/java/org/apache/cxf/attachment/AttachmentDeserializer.java

```diff
@@ -456,11 +455,8 @@ private void addHeaderLine(Map<String, List<String>> heads, StringBuilder line)
             }
             value = line.substring(separator);
         }
-        List<String> v = heads.get(name);
-        if (v == null) {
-            v = new ArrayList<>(1);
-            heads.put(name, v);
-        }
+        List<String> v = heads.computeIfAbsent(name, k -> new ArrayList<>(1));
+        heads.put(name, v);
```

Comment:
additional put is unnecessary

Reply:
@amarkevich  .  many thanks for the feedback. I actually missed the javadoc for Map#computeIfAbsent  
which states:
 The mostcommon usage is to construct a new object serving as an initialmapped value or memoized result, as in: 
 map.computeIfAbsent(key, k -> new Value(f(k)));

additional put has been removed now. with thanks